src/core/libmaven/mzSample.h

```diff
@@ -711,6 +711,15 @@ class mzSample
 
     static bool compSampleSort(const mzSample *a, const mzSample *b) { return mzUtils::strcasecmp_withNumbers(a->sampleName, b->sampleName); }
 
+    /**
+    * @brief Compare injection time (in epoch seconds) of two samples
+    * @param a object of class mzSample
+    * @param b object of class mzSample
+    * @return True if mzSample a has lower injection time than mzSample b
+    */
+    static bool compInjectionTime(const mzSample *a, const mzSample *b) { return a->injectionTime < b->injectionTime; }
```

Comment:
why this has to be a static function ?

Reply:
@rish9511 So that we can use the function without object (like mzSample::compInjectionTime)